protected/jobs/accessRequests/ImplicitlyGrantedNotificationJob.php

```diff
@@ -27,9 +27,14 @@ public function getPartial(): bool
 
     public function jsonSerialize(): array
     {
-        return [
-            'accessRequestId' => $this->accessRequestId,
+        /**
+         * @todo When we upgrade to php 8.1 use spread operator
```

Comment:
At what version are we? You are using the spread operator on https://github.com/HeRAMS-WHO/herams-backend/pull/947/files#diff-5e439c58860b64b6d26b0b918235407aa1b9a729f4bb57eb9556f4d4d838c4a0R28